readme.txt

```diff
@@ -34,12 +34,24 @@ In the plugin settings you can edit the title of the plugin, and enable or disab
 == Screenshots ==
 
 1. This is the meta box right to the post editor.
-1. This is the posts page.
-1. These are the widget options.
-1. This is the control panel in frontend for logged in users.
-1. This is the widget in frontend for not logged in users.
-1. This is the statistics page which displays the top 25 posts.
-1. This is the settings page.
+Post Worktime Logger will add a new interactive box to your edit screen, allowing you to keep track of how much time you spend on your alterations.
```

Comment:
Oh, sorry about that! I'll remove **all** that breaks right now, and you can see if it works :smile: